shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -85,13 +87,15 @@ static bool ClipRRectContainsPlatformViewBoundingRect(const SkRRect& clip_rrect,
 
 std::shared_ptr<FlutterPlatformViewLayer> FlutterPlatformViewLayerPool::GetLayer(
     GrDirectContext* gr_context,
-    const std::shared_ptr<IOSContext>& ios_context) {
+    const std::shared_ptr<IOSContext>& ios_context,
+    MTLPixelFormat pixel_format) {
   if (available_layer_index_ >= layers_.size()) {
     std::shared_ptr<FlutterPlatformViewLayer> layer;
     fml::scoped_nsobject<UIView> overlay_view;
     fml::scoped_nsobject<UIView> overlay_view_wrapper;
 
-    if (!gr_context) {
+    bool impeller_enabled = !!ios_context->GetImpellerContext();
```

Comment:
`!!`? `bool impeller_enabled = ios_context->GetImpellerContext();`?

Reply:
conversion is explicit for this type

Reply:
It should be going through the cast operator fine: https://en.cppreference.com/w/cpp/memory/shared_ptr/operator_bool

Reply:
Oh wait, that's explicit too.  `!!` is weird.  I don't think there is a rule against it though so I can't ask for something else, lgtm.  It seems like an implicit cast is preferred to `!!`.